### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by tegk**
I reviewed, cleaned and refactored some parts of the code.
Will do more work when I find more time.

### CODE REVIEW

**Comment by tegk** [comment]
Sorry that the checks failed, will fix tomorrow.

File: goguerrilla.conf.api [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "servers" : [
+        {
+            "is_enabled" : true,
+            "host_name":"mail.guerrillamail.com",
+            "max_size": 100017,
+            "timeout":160,
+            "listen_interface":"127.0.0.1:2526",
+            "max_clients": 2,
+			"tls" : {
+ 				"private_key_file":"config_test.go",
```
Comment (lord-alfred) [inline goguerrilla.conf.api]: tabs, not spaces

File: tests/pidfilex.pid [hunk 2]
```diff
@@ -0,0 +1 @@
+30673
```
Comment (lord-alfred) [inline tests/pidfilex.pid]: maybe need gitignored?

**Comment by lord-alfred** [comment]
files "tests/testlog" and "tests/testlog2" empty

**Pull request closed without merging**